src/JContainers/src/skse/skse.cpp

```diff
@@ -158,11 +158,9 @@ struct real_api : public skse_api
 
     FormId resolve_handle (FormId handle) override
     {
-        std::uint64_t out = 0,
-                      in  = static_cast<std::uint64_t> (handle);
-        if ((in & 0xFF000000u) == 0xFF000000u)
-            return handle; // dynamic form - return untouched
-        return g_serialization->ResolveHandle (in, &out) ? (FormId) out : FormId::Zero;
+		auto oldID = static_cast<UInt32>(handle);
```

Comment:
Truncating 64 to 32, is this the way?

Reply:
Hmm... yes, true. Thats what you get when mixing types and names. `FormId` is 32-bit.